host/ProtocolGateway.Host.Fabric/Scripts/Deploy-FabricApplication.ps1

```diff
@@ -0,0 +1,220 @@
+﻿<#
+.SYNOPSIS 
+Deploys a Service Fabric application type to a cluster.
+
+.DESCRIPTION
+This script deploys a Service Fabric application type to a cluster.  It is invoked by Visual Studio when deploying a Service Fabric Application project.
+
+.NOTES
+WARNING: This script file is invoked by Visual Studio.  Its parameters must not be altered but its logic can be customized as necessary.
+
+.PARAMETER PublishProfileFile
+Path to the file containing the publish profile.
+
+.PARAMETER ApplicationPackagePath
+Path to the folder of the packaged Service Fabric application.
+
+.PARAMETER DeloyOnly
```

Comment:
I changed it but so you know that is the default script file generated by visual studio for service fabric (auto gen). I will notify the SF team that this typo is there